### PR TITLE
Replace Root AMS serializers with JSONAPI serializers - Part 6

### PR DESCRIPTION
**Pull request by stevenjcumming**
## Summary

Switched the following to JSONAPI::Serializer:

- ContactSerializer
- DependentsSerializer
- DependentsVerificationsSerializer

**_`REMOVED`_** the following serializers: 
- DependentsApplicationSerializer - didn't appear to be used anywhere

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/87047

## Testing done

- [x]  No tests added or updated
	
## Acceptance criteria

- [x] Each serializer uses jsonapi-serializer 
- [x] Each serializer has 100% coverage

### CODE REVIEW

File: app/serializers/dependents_application_serializer.rb
Comment (rmtolmach) [inline app/serializers/dependents_application_serializer.rb:3]: Do you want a review from @department-of-veterans-affairs/vfs-authenticated-experience-backend?

**Review by rmtolmach — DISMISSED** [review]
LGTM, but not sure if you wanted another review from @department-of-veterans-affairs/vfs-authenticated-experience-backend

File: app/serializers/dependents_application_serializer.rb
Comment (stevenjcumming) [reply app/serializers/dependents_application_serializer.rb:3]: I think they should. If it's being used, then I can delete it (and the spec)

**Comment by stevenjcumming** [comment]
talked with @tpharrison about DependentsApplicationSerializer and it's not being used, so I deleted it.

**Pull request merged**
